### DISCUSSION TITLE
1-2. 회사가 정의하는 개발자

### DISCUSSION THREAD

**Discussion by imakerjun** [ssac-dev/yozm#4] [1장 이런 개발자가 되고 싶어]
# 1-2. 회사가 정의하는 개발자
## 1. 내가 자주 사용하는 서비스 기업의 핵심 가치 찾기
회사의 핵심 가치는 구성원들이 일을 하는 핵심 이유이자 목표이기 때문에 매우 중요합니다. 자주 애용하는 서비스 기업의 홈페이지나 채용 페이지를 방문하여 그 기업의 핵심 가치를 찾아보세요.

## 2. 개발자 콘퍼런스에서 주최한 기업의 다음 방향성을 드러내는 서비스 찾기
개발자 콘퍼런스는 주최 기업의 최신 기술을 뽐내기 가장 좋은 장 중 하나입니다. 그래서 주최 기업의 방향성, 미래성이 드러내는 서비스를 쉽게 접할 수 있습니다. 그 기업에서 뽐내는 서비스를 찾아보고 또 직접 접해보세요.

## 3. 기업의 블로그나 유튜브를 통해 신규 입사자를 위한 온보딩 콘텐츠 찾기
기업의 서비스 홍보 및 채용을 위해 기업 전용 블로그나 유튜브를 운영하는 경우가 많습니다. 검색을 통해 관심 기업의 블로그나 유튜브를 찾아보고, 블로그와 유튜브의 컨텐츠를 통해 신규 입사자를 위한 온보딩 콘텐츠를 찾아보세요.

**Comment by baobabnamu** [comment]
# 1. 내가 자주 사용하는 서비스 기업의 핵심 가치 찾기
제가 최근에 자주 사용하는 서비스는 밀리의 서재입니다. 밀리의 서재는 KT의 핵심 가치를 따르고 있습니다.  

KT 그룹의 핵심 가치는 다음과 같습니다. :)  

[고객]  
고객의 니즈 충족과 문제 해결을 위해 치열하게 고민하고 새로운 고객 경험을 제시합니다.  

[역량] 
고객의 문제를 해결하고 고객이 원하는 혁신을 가장 잘할 수 있도록 전문성을 높입니다.  

[실질]
우리의 본업인 통신과 ICT를 단단히 하고 화려한 겉모습보다 실질적인 성과를 추구합니다.  

[화합]
다름을 인정하되 서로 존중하고 합심해 함께 목표를 이뤄 갑니다.

**Comment by heeji289** [comment]
# 1-2. 회사가 정의하는 개발자
## 1. 내가 자주 사용하는 서비스 기업의 핵심 가치 찾기
어렵고, 불편하고, 멀게 느껴지는 금융이 아닌 **누구에게나 쉽고 상식적인 금융을 만드는 것**이 토스팀의 존재 이유입니다.

## 2. 개발자 콘퍼런스에서 주최한 기업의 다음 방향성을 드러내는 서비스 찾기
server-driven UI ??

## 3. 기업의 블로그나 유튜브를 통해 신규 입사자를 위한 온보딩 콘텐츠 찾기
온보딩 그냥 따라하기만 되도록 구성해놓는 듯.

(우선 메이트 제도를 운영하고 있는데, 실제로 가장 가까이 함께 일할 프론트엔드 팀원을 배정해드리고 온보딩시 필요한 체크리스트를 제작하고 있어요. 그리고 체크리스트에 따라 개인화된 온보딩 로그를 추가 제작 후 전달하고 있습니다. 계정 세팅부터 프론트엔드 개발 지식 그리고 배포와 모니터링 방법까지 토스 적응에 필요한 A to Z를 알려드리고 있죠. 더 나은 온보딩 프로그램 발전을 위해 입사 3~4주차 때 온보딩 과정에 대한 피드백을 줄 수 있는 창구도 만들어 놨어요.)

**Comment by Bigstargithub** [comment]
## 내가 자주 사용하는 서비스 기업의 핵심 가치 찾기
유튜브 회사의 구성원들은 모든 사람들에게 세상 모든 모습과 목소리를 들려주는 핵심가치를 가지고 일하고 있습니다.

## 개발자 컨퍼런스에서 주최한 기업의 다음 방향성을 드러내는 서비스 찾기
우아콘: https://woowacon.com
인프콘: https://www.inflearn.com/conf/infcon-2024/

## 기업의 블로그나 유튜브를 통해 신규 입사자를 위한 온보딩 콘텐츠 찾기
토스 온보딩 프로그램 관련 기사: https://toss.im/career/article/toss-onboarding

**Comment by gogo1414** [comment]
## 1-2. 회사가 정의하는 개발자
### 1. 내가 자주 사용하는 서비스 기업의 핵심 가치 찾기
- 제가 자주 사용하는 서비스는 "**카카오맵**" 입니다.
- 카카오 기업의 서비스 가치는 "기술로 세상을 더 즐겁게"라는 핵심 가치를 추구합니다.
- 카카오맵의 서비스 가치는 "다양한 상황과 환경에 놓인 사용자들의 서로 다른 일상을 빈틈없이 메워줄 수 있는 것"을 추구합니다.

### 2. 개발자 콘퍼런스에서 주최한 기업의 다음 방향성을 드러내는 서비스 찾기
- [카카오 개발자 컨버런스 2024](https://www.kakaocorp.com/page/detail/10989)를 보며 알 수 있던 것은 카카오 서비스에 AI를 도입하고 있는 모습입니다. 확실히 인공지능 기술이 화두가 되고 있는 만큼 카카오라는 대기업에서도 인공지능을 도입하는 모습을 보여주고 있는 것 같습니다.

### 3. 기업의 블로그나 유튜브를 통해 신규 입사자를 위한 온보딩 콘텐츠 찾기
- [2023 카카오 신입공채 온보딩](https://tech.kakao.com/posts/576)

**Comment by CHPark00** [comment]
**### 1. 내가 자주 사용하는 서비스 기업의 핵심 가치 찾기**
제가 요즘 사용하고 있는 서비스는 "카카오톡"입니다.
제가 찾은 카카오톡을 비롯한 카카오 그룹의  핵심 가치는 1.혁신 2.성장 3.사람에 대한 이해입니다.

### **3.기업의 블로그나 유튜브를 통해 신규 입사자를 위한 온보딩 콘텐츠 찾기**
카카오 유튜브는 있지만 회사 규모에 비해 활동이 크게 이루어지고 있지 않다고 생각이 듭니다.(크게 홍보는 안하는 듯 싶네요)

링크: https://youtube.com/@kakaobrandmedia?si=yarqsKdjne6gtyJb

**Comment by dongjooyun** [comment]
# 1-2. 회사가 정의하는 개발자
## 1. 내가 자주 사용하는 서비스 기업의 핵심 가치 찾기

- 기업명: 비바리퍼블리카
- [참고 링크](https://toss.oopy.io/c718519a-671e-475c-9c18-e49384d29636)

핵심 가치(영문) | 핵심 가치(한글) | 설명 요약
-- | -- | --
Mission over Individual | 개인 목표보다 팀의 미션 우선 | 개인의 목표보다 토스팀의 미션을 우선한다
Aim Higher | 더 높은 수준을 추구 | 기대를 뛰어넘는, 더 높은 기준과 성과를 지향
Focus on Impact | 임팩트에 집중 | 하면 좋을 10가지보다, 실제로 임팩트를 만드는 일에 집중
Question Every Assumption | 모든 가정에 근원적 질문 | 기존의 방식이나 전제에 끊임없이 의문을 제기
Execution over Perfection | 완벽보다 실행 | 완벽을 추구하기보다 빠른 실행과 실질적 결과를 중시
Learn Proactively | 주도적 학습 | 스스로 배우고 성장하는 자세 강조
Move with Urgency | 신속하게 움직임 | 빠른 실행과 의사결정, 민첩한 행동
Ask for Feedback | 피드백을 자주 구함 | 동료와의 활발한 피드백을 통해 솔직하고 투명한 커뮤니케이션 추구


## 2. 개발자 콘퍼런스에서 주최한 기업의 다음 방향성을 드러내는 서비스 찾기
- 가장 최근에 개최된 토스 개발자 컨퍼런스: [SLASH 24](https://toss.im/slash-24)
- 비바리퍼블리카의 방향성: “한계 없는 도전”, “실패를 두려워하지 않는 혁신”
- 뽐낸 서비스: 본인확인(금융, 공공, 민간 토스 인증), 토스 앱(통합 금융·생활 플랫폼), 계열사 확장 등

## 3. 기업의 블로그나 유튜브를 통해 신규 입사자를 위한 온보딩 콘텐츠 찾기
토스 기술블로그에 [디자이너 온보딩 프로세스에 관한 글](https://toss.tech/article/designer-onboarding)은 있는데, 개발자 온보딩 관련 글은 없었다.
토스 채용 페이지에 합류 후 프로세스에 대한 안내에 [온보딩 세션](https://toss.im/career/joining-guide)이 언급되어있다.
토스 유튜브 쇼츠 중에 [신규 입사자를 위한 조언](https://www.youtube.com/watch?v=U5Xr2bPrUTM)이 있다.
